internal/resources/oncall/data_source_integrations.go

```diff
@@ -0,0 +1,49 @@
+package oncall
```

Comment:
This file should be named `data_source_integration.go`

Reply:
Renamed them